### PR TITLE
Added Recipe for Tika

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/tika/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://pypi.io/packages/source/t/tika/tika-{{ version }}.tar.gz
+  md5: 6f5e508edf645e43f1bd0c41a5b47202
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - tika-python = tika.tika:main
+
+  number: 0
+  # script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipes/tika/meta.yaml]: I guess that you are planning on un-commenting this line :wink:

File: recipes/tika/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    # entry points work.
+
+    - tika-python --help
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
```
Comment (ocefpaf) [inline recipes/tika/meta.yaml]: You can remove all the `conda-skeleton` boilerplate comments.

Comment (pmlandwehr) [inline recipes/tika/meta.yaml]: Hah! Thought I'd gotten those.

**Comment by ocefpaf** [comment]
Thanks @pmlandwehr!

**Pull request merged**